src/Hbar.ts

```diff
@@ -59,35 +69,46 @@ export class Hbar {
     private readonly _tinybar: BigNumber;
     private readonly _unit: HbarUnit;
 
-    private constructor(tinybar: BigNumber, unit: HbarUnit = HbarUnit.Hbar) {
-        this._tinybar = tinybar;
-        this._unit = unit;
+    public constructor(amount: number | BigNumber | string) {
```

Comment:
Instead of divide/mulitply to make a new Hbar from tinybar. Follow this pattern,

```js
function fromTinybar(newTinybar: number): Hbar {
    var newHbar = new Hbar(0);
    newHbar._tinybar = newTinybar;
    return newHbar;
}
```